research-app/src/App.vue

```diff
@@ -76,6 +118,8 @@ import * as screenfull from "screenfull";
 import 'vue-select/dist/vue-select.css';
 import { Component, Prop, Watch } from "vue-property-decorator";
 import { fmtDegLat, fmtDegLon, fmtHours } from "@wwtelescope/astro";
+import WWTResearchAppModule from "./store";
+import { mapMutations, mapState } from "vuex";
```

Comment:
For sequences of imports like these, I generally try to sort them in the following rough categories:

- Common external libraries
- Application-specific external libraries
- Internal dependencies

So I'd put the vuex import farther up, here.